utils/nft.ts

```diff
@@ -4,30 +4,6 @@ import type { NFTWithMetadata } from '@/composables/useNft'
 export const parseNftAvatar = async (
   entity: EntityWithMeta,
 ): Promise<string> => {
-  const { urlPrefix } = usePrefix()
-  const meta = await getNftMetadata(
-    entity as unknown as NFTWithMetadata,
-    urlPrefix.value,
-  )
+  const meta = await getNftMetadata(entity as unknown as NFTWithMetadata)
   return meta.image
 }
-
-const trimLeadingZeros = (of: number | string) => {
-  const trimmedString = of.toString().replace(/^0+(?=\d)/, '')
-
-  return typeof of === 'number' ? Number(trimmedString) : trimmedString
-}
-
-const suffixRegex = /#\d+$/
-export const nameWithIndex = (name: string = '', sn?: string) => {
-  if (!name || !sn) {
-    return name
-  }
-
-  // display sn if nftId is less than 4 digits
-  if (sn.length >= 4) {
-    return name
-  }
-
-  return suffixRegex.test(name) ? name : `${name} #${trimLeadingZeros(sn)}`
-}
```

Comment:
why? There was an issue before that it wants to show the `#sn` number on the name of nft.

Reply:
Refer to my comment here: [PR #10934 Comment](https://github.com/kodadot/nft-gallery/pull/10934#issuecomment-2335128843)

Just return it as it is from token metadata. I guess that one is for old drops collection, isn't it? If yes, we can update old drops metadata instead. Something like this: [Issue #217](https://github.com/kodadot/private-workers/issues/217)

Because the indexer doesn't return the proper name in some tokens, AssethHub needs an additional request. Here are the changes: [`c2fb564` (#10934)](https://github.com/kodadot/nft-gallery/pull/10934/commits/c2fb564743dec98c1a720b8f7e74903b83e48926)